libs/utils/utils.js

```diff
@@ -782,3 +783,11 @@ export function loadLana(options = {}) {
   window.addEventListener('error', lanaError);
   window.addEventListener('unhandledrejection', lanaError);
 }
+
+export function debounce(func, timeout = 300) {
```

Comment:
On second thought, adding the `debounce` function to `utils/utils.js` might not be such a good idea since we're trying to keep this file as small as possible.

@auniverseaway , wdyt?

Reply:
I know I wasn't asked, but I agree with you @honstar 
that code doesn't belong in utils.

Reply:
I'm just seeing this now. This should not have been allowed in `utils.js`. There's no reason a visitor should pay for those bytes unless they need them. To my knowledge, no other function needs this.

@vhargrave @honstar @Axelcureno can one of you PR to change the location?

I will make a note somewhere about the criteria by which something should be allowed into utils.js.

When we first built utils.js, we had some spare bytes where we put things in there for simplicity sake. As utils.js has grown, we need to get back to it being a smaller file where every line should be justified. We set a bad example by having things like `utf8ToB64` in there.

We have since added more critical items like htmlPostFix and localizeLink and we should be kicking out non-critical functions (like @narcis-radu did with loadDelayed).

If memory serves, we should be able to get it down to 600 lines without making things less readable.

Reply:
@auniverseaway done - created a new PR https://github.com/adobecom/milo/pull/930